src/master/filesystem_operations.cc

```diff
@@ -2019,11 +2023,14 @@ uint8_t fs_writeend(uint32_t inode, uint64_t length, uint64_t chunkid, uint32_t
 			return SAUNAFS_ERROR_EPERM;
 		}
 		if (length > p->length) {
-			fsnodes_setlength(p, length);
+			bool eraseFurtherChunks = false;
```

Comment:
Same suggestion here about explaining the reason of `eraseFurtherChunks = false`.

Reply:
Applied.